BuildAddOn.py

```diff
@@ -208,7 +228,11 @@ def BuildAddOn (addOnName, platformName, additionalParams, workspaceRootFolder,
 
     # Add params to configure cmake
     projGenParams = GetProjectGenerationParams (workspaceRootFolder, buildPath, addOnName, platformName, devKitFolder, version, languageCode, additionalParams)
-    projGenResult = subprocess.call (projGenParams)
+    if quiet:

```

Comment:
Why do we need a quiet mode?

Reply:
Quiet mode was added by Somorjai Ákos, but it only hid 4 lines of the total output.

Reply:
@asomorjai do we really need this?

Reply:
Yes, on the Mac it significantly reduced the output.

Reply:
@asomorjai when using quiet mode, is there any output that should still be visible?
Because my current solution completely hides the output